src/Form/DatasetSubmissionType.php

```diff
@@ -69,37 +69,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'expanded' => true,
                 'multiple' => false,
             ))
-            ->add('datasetFileUri', Type\HiddenType::class, array(
```

Comment:
This is being used in `templates/DatasetSubmission/file-submit.html.twig`

Reply:
I need to do a grep on templates and javascript files